src/plugins/github/parseReferences.js

```diff
@@ -98,6 +99,47 @@ function findGithubUrlReferences(textBlock: string): ParsedReference[] {
   }));
 }
 
+function findCommitReferences(textBlock: string): ParsedReference[] {
+  const hashReferences = findAllMatches(
+    /(?:\W|^)([a-fA-F0-9]{40})(?=\W|$)/gm,
```

Comment:
Git is in the process of moving from SHA-1 to SHA-256, which changes the
hex digest from 40 characters to 64. Do you want to modify your regular
expression, changing `{40}` to `{40,}`, to be forward-compatible?

See [`git/git:Documentation/technical/hash-function-transition.txt`[1]
for more details.

[1]: https://github.com/git/git/blob/1d4361b0f344188ab5eec6dcea01f61a3a3a1670/Documentation/technical/hash-function-transition.txt